shex/sio-model/diseaseHistoryAndDiagnosisShape.shex

```diff
@@ -1,27 +1,43 @@
 PREFIX : <http://purl.org/ejp-rd/cde/v020/shex/>
 PREFIX obo: <http://purl.obolibrary.org/obo/>
 PREFIX sio: <http://semanticscience.org/resource/>
-PREFIX dct: <http://purl.org/dc/terms/>
 PREFIX xsd: <http://www.w3.org/2001/XMLSchema#>
+PREFIX prov: <http://www.w3.org/ns/prov#>
+PREFIX loinc: <http://purl.bioontology.org/ontology/LNC/>
 
 
 :personShape IRI {
-  a [obo:NCBITaxon_9606];
-  sio:SIO_000228 @:personRoleShape
+  a [sio:SIO_000498];
+  sio:SIO_000228 @:personRoleShape;
+  sio:SIO_000217 @:ageAtDiagnosisQualityShape;
+  sio:SIO_000217 @:symptomOnsetQualityShape
 }
 
 :personRoleShape IRI {
   a [obo:OBI_0000093];
-  sio:SIO_000356 @:medicalDiagnosisProcessShape
+  sio:SIO_000356 @:processShape +
```

Comment:
"+" means one or more: one role should have at least one process.

Reply:
Sometime to check with @markwilkinson . In the example turtle I connected multiple process to same role. For now we use this pattern we might change it based on next week's review.